src/components/Preferences/Preferences.tsx

```diff
@@ -104,6 +125,16 @@ export function Preferences() {
             <ResourcesValidationSettings />
           </div>
         </VerticalTabs.Content>
+        <VerticalTabs.Content id={3}>
+          <div>
+            {isExtensibilityEnabled && (
+              <Toggle
```

Comment:
Toggle name could provide a little bit misunderstanding, could you change this for something like `DevelopmentSettings` or `DebugonataSettings` or something like this as we have for namespaceSetting etc